src/components/Navbar/AlarmModal.module.css

```
@@ -0,0 +1,111 @@
+@values tablet, mobile from '@/styles/breakpoints.css';
+
+@keyframes slidein {
+  from {
+    transform: translateY(-100%);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+@keyframes slideout {
+  from {
+    transform: translateY(0);
+    opacity: 1;
+  }
+  to {
+    transform: translateY(-100%);
+    opacity: 0;
+  }
+}
+
+@keyframes slideinpc {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+@keyframes slideoutpc {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
+
+.alarmModalContainer {
+  position: absolute;
+  border: 0.1rem solid var(--grayDD);
+  border-radius: 0.8rem;
+  top: 6rem;
+  right: 17rem;
+  background-color: var(--greenCE);
+  width: 36.8rem;
+  height: auto;
+  padding: 2.4rem 2rem;
+  animation: slideinpc 0.3s ease-out;
+
+  @media mobile {
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 80vh;
+    border-radius: 0 0 0.8rem 0.8rem;
+    animation: slidein 0.3s ease-out;
+  }
+}
+
+.alarmModalCloseContainer {
+  composes: alarmModalContainer;
+  animation: slideoutpc 0.3s ease-out;
+  @media mobile {
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 80vh;
+    border-radius: 0 0 0.8rem 0.8rem;
+    animation: slideout 0.3s ease-out;
+  }
+}
+
+.alarmModalHeader {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1.6rem;
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.alarmModalContentContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.8rem;
+}
+
+.alarmModalContentWrapper {
+  background-color: var(--whiteFF);
+  width: 32.8rem;
+  height: auto;
+  border-radius: 0.8rem;
+  padding: 1.6rem 1.2rem;
+
+  @media mobile {
+    width: 100%;
+  }
+}
+
+.alarmModalContentHeader {
+  display: flex;
+  justify-content: space-between;
+}
+
+.alarmModalContentTime {
+  margin-top: 0.4rem;
+  color: var(--grayA4);
+}
```

font-size, line-height가 적용되지 않은 것 같습니다. 
```suggestion
.alarmModalContentTime {
  margin-top: 0.4rem;
  color: var(--grayA4);
  font-size: 1.2rem;
  line-height: 1.6rem;
}
```